tool/docker/fedora/Dockerfile

```diff
@@ -31,27 +31,28 @@ RUN git clone --depth 1 https://github.com/graalvm/mx.git
 ENV PATH=$PATH:/build/mx
 
 # Build a JDK with JVMCI
-RUN git clone --depth 1 https://github.com/graalvm/graal-jvmci-8.git
+RUN git clone https://github.com/graalvm/graal-jvmci-8.git
```

Comment:
Why removing the `--depth 1`? It makes building those docker images much faster.

Reply:
The point of the tool images was that you can develop in them, making changes and pushing fixes.

Reply:
I see, sounds good then.
FWIW there is a way to expand a shallow git clone with `git fetch --unshallow` but that's not intuitive.

Reply:
Apparently that absolutely hammers Git servers for some reason - GitHub were complaining about people doing shallow clones and then unshallowing later.